include/roaring/roaring64.h

```diff
@@ -15,6 +15,7 @@ namespace api {
 
 typedef struct roaring64_bitmap_s roaring64_bitmap_t;
 typedef struct roaring64_leaf_s roaring64_leaf_t;
+typedef struct roaring64_iterator_s roaring64_iterator_t;
```

Comment:
If we're hiding the iterator type, maybe we should have a `roaring64_iterator_reinit(roaring64_iterator_t *, const roaring64_bitmap_t *)`, to allow reusing an iterator with multiple bitmaps (or after modifications are made to the same bitmap) without having to allocate again